src/app/sharedUI/screens/Artwork/Artwork.tsx

```diff
@@ -129,8 +129,8 @@ export const Artwork = () => {
                   ? "Currently in 1 album"
                   : `Currently in ${numberOfAlbumsIncludingArtwork} albums`
               }
-              onPress={() =>
-                navigation.navigate("AddArtworkToAlbum", { slug, contextArtworkSlugs })
+              navigateTo={() =>
```

Comment:
are we having wrong types here? this complains with `navigateTo`, and it is ok with `onPress` 🤔.
```suggestion
              onPress={() =>
```